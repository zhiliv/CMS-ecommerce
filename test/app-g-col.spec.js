import { shallowMount } from '@vue/test-utils'
import Row from '@/components/app/g-col/g-col.vue'

describe('app-g-col.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Row, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        gCol: true,
        gColSm: 'auto',
        gColMd: true,
        gColXl: 4,
        gColXxl: 6,
        offset: 3,
        offsetSm: 2,
        offsetMd: 3,
        offsetXl: 4,
        offsetXxl: 6,
        classes: 'simple',
        styles: 'background: white',
        id: 'indent'
      },
    })
  })
  it('Содержимое html для app-g-col ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
