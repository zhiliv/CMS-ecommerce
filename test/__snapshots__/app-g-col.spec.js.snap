// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-g-col.js Содержимое html для app-g-col  1`] = `
<div
  class="g-col g-col-sm-auto g-col-md g-col-xl-4 g-col-xxl-6 offset-3 offset-sm-2 offset-md-3 offset-xl-4 offset-xxl-6 simple"
  id="indent"
  style="background: white;"
/>
`;
